tools/report-converter/codechecker_report_converter/analyzers/analyzer_result.py

```diff
@@ -148,10 +149,13 @@ def _write(
         analyzer_info = AnalyzerInfo(name=self.TOOL_NAME)
         for file_path, file_reports in file_to_report.items():
             source_file = os.path.basename(file_path)
+            file_hash = hashlib.md5(file_path.encode(errors='ignore')) \
```

Comment:
You can add a new test case or extend the existing one here (`test_metadata`): https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/tests/functional/cmdline/test_cmdline.py#L28

In the test case you have to check that multiple plist files are generated.

For this you also have to modify couple of files here: https://github.com/Ericsson/codechecker/tree/master/tools/report-converter/tests/functional/cmdline/test_files

Add a new file `report-converter/tests/functional/cmdline/test_files/files/b/simple.go` with the following content:
```go
package pgk

type T2 int

var Y2, Z2 int
```

Modify the Makefile here:
https://github.com/Ericsson/codechecker/blob/7df76b0429e1bcc4f242c01dae36c92bffc830ef/tools/report-converter/tests/functional/cmdline/test_files/Makefile#L2

to 
```makefile
all:
	golint ./files/simple.go files/b/simple.go > simple.out
```

Modify the https://github.com/Ericsson/codechecker/blob/master/tools/report-converter/tests/functional/cmdline/test_files/simple.out to this:
```
./files/simple.go:3:6: exported type T should have comment or be unexported
./files/simple.go:5:5: exported var Z should have its own declaration
./files/b/simple.go:3:6: exported type T2 should have comment or be unexported
./files/b/simple.go:5:5: exported var Z2 should have its own declaration
```